requirements.in

```diff
@@ -1,6 +1,6 @@
 beautifulsoup4
 boto3
-Django==2.2.14
+Django==2.2.15
```

Comment:
let's make this 2.2.16 (security update)

Reply:
```suggestion
Django==2.2.16
```